config/config.go

```diff
@@ -287,6 +289,7 @@ var defaultConf = Config{
 		MetricsInterval: 15,
 	},
 	Performance: Performance{
+		MaxMemory:           0,
```

Comment:
If the configed `MaxMemory` is larger than the total memory of the machine, should we adjust this value to the total memory of that machine?